package.json

```diff
@@ -4,64 +4,63 @@
   "private": true,
   "type": "module",
   "dependencies": {
-    "@azure/msal-browser": "^3.10.0",
-    "@azure/msal-react": "^2.0.12",
-    "@emotion/react": "^11.11.3",
-    "@emotion/styled": "^11.10.8",
-    "@mui/icons-material": "^5.15.10",
-    "@mui/material": "^5.15.10",
-    "@mui/x-date-pickers": "^6.19.4",
-    "add-to-calendar-button": "^2.5.10",
-    "caniuse-lite": "^1.0.30001589",
-    "chart.js": "^2.9.4",
-    "cropperjs": "^1.6.1",
-    "date-fns": "^3.3.1",
+    "@azure/msal-browser": "^3.16.0",
```

Comment:
@russtuck I think you may be missing the nuance of how caret ranges handle `0.x` versions. `node-semver` is not at odds with semver.org, and actually caret ranges are slightly looser with presumed non-breaking changes than what semver.org specifies.

Specifically, semver.org says that:
> 4. Major version zero (0.y.z) is for initial development. Anything MAY change at any time. The public API SHOULD NOT be considered stable.

Whereas node-semver says of it's handling of caret ranges (emphasis mine):
> Allows changes that do not modify the left-most non-zero element in the [major, minor, patch] tuple. In other words, this allows **_patch and minor_** updates for versions 1.0.0 and above, **_patch updates for versions 0.X >=0.1.0_**, and no updates for versions 0.0.X.
>
>Many authors treat a 0.x version as if the x were the major "breaking-change" indicator.

Notice that, for example, the caret range `^18.3.1` would allow upgrading `18.3.5` or `18.4.0`, which agrees with semver.org (and which you may have been misunderstanding). Furthermore, the caret range `^0.4.3` would allow upgrading to `0.4.4`, whereas according to semver.org "Major version zero (0.y.z) is for initial development. Anything MAY change at any time.".

I think the crucial point is that, as `node-semver` points out, "Many authors treat a 0.x version as if the x were the major "breaking-change" indicator", so caret ranges protect you from upgrading `0.4.3` to `0.5.0`, because `0.4 -> 0.5` is expected to have breaking changes.

Overall, I would say that npm's default way of specifying package versions is helpful in stating which package was intentionally installed (i.e. at the time this was last carefully reviewed and manually upgraded, we installed "18.3.1"), while allowing upgrades to non-breaking versions automatically (and we currently have "18.5.1" installed).

Does that address your concerns @russtuck?